.semaphore/semaphore.yml

```diff
@@ -24,15 +24,16 @@ blocks:
       - name: Check fedora-32
         commands:
           - export TEST_OS=fedora-32
-          - bots/image-customize -v -i cockpit-ws -i `pwd`/cockpit-session-recording*.noarch.rpm -s `pwd`/test/vm.install $TEST_OS
+          - bots/image-customize -v -i cockpit-ws -i cockpit-packagekit -i `pwd`/cockpit-session-recording*.noarch.rpm -s `pwd`/test/vm.install $TEST_OS
+          - bots/image-customize -v -r "usermod -u 981 tlog || true" $TEST_OS
```

Comment:
Is this line usermod added twice?

Reply:
No that shouldn't have been there.  should be gone in latest push.